### DISCUSSION TITLE
Great minds starred us 20K times(We’re so thankful)  🥳

### DISCUSSION THREAD

**Discussion by chandrevdw31** [mindsdb/mindsdb#8817] [Announcements]
MindsDB is celebrating reaching 20K Stars! We are extremely grateful to each and every person who has starred our repo and would like to show our appreciation.

In the coming 3 weeks, we’ll be giving away 10 **limited edition T-shirts** each week to selected people who star MindsDB. Put your name in the hat by starring MindsDB’s [Github repo](link to https://github.com/mindsdb/mindsdb)

The 10 lucky winners will be announced weekly in this comments thread and posted on our Twitter page, so be sure to check back.

If you’ve been selected, please reach out to us on Community Slack to collect your T-shirt. You can DM Chandre directly for assistance.

MindsDB thanks you for your continuous support.

**Comment by pravintargaryen** [comment]
@chandrevdw31. What about already starred people?

**Reply by chandrevdw31** [reply]
Hey @pravintargaryen yes they also stand a chance to win

**Comment by chandrevdw31** [comment]
MindsDB is giving away **limited edition T-shirts** to 10 lucky people each week. Make sure to tune in each week to see who the lucky winners are. Check out our awesome **limited edition T-shirt** below.
![image](https://github.com/mindsdb/mindsdb/assets/32901682/d5bbd026-4149-4dfd-8a1b-f889d5080090)
![image](https://github.com/mindsdb/mindsdb/assets/32901682/30768dd6-6240-4518-a2a5-252df04bf095)

**Reply by morgankar** [reply]
Looks great!

**Reply by chandrevdw31** [reply]
Glad you like it!

**Comment by chandrevdw31** [comment]
**Massive congrats to the contributors who starred our GitHub repo and helped push us over the 20K mark!**

We’re thrilled to present this **limited-edition T-shirt** as a token of our gratitude.

Your contributions truly make an open-source project like MindsDB great.

**20 000th Github Stargazer: Leon Gu([lebrosoft](https://github.com/lebrosoft))**

Additional winners for this week:

- Parthiv Makwana https://github.com/parthiv11
- Jacopo-minniti https://github.com/jacopo-minniti
- Bispaul https://github.com/bispaul
- Pranavvp16 https://github.com/pranavvp16
- HahaBill https://github.com/HahaBill
- Derrick Hammer https://github.com/pcfreak30
- Hju Kneyck Flores https://github.com/huenique
- DavidGOrtega https://github.com/DavidGOrtega
- Viktor Sokolov https://github.com/victory-sokolov
- Sekhar-Kumar-Dash  https://github.com/Sekhar-Kumar-Dash 

Please contact Chandre on MindsDB Slack to claim your T-shirt if your name is mentioned above..Make sure to check out this comments thread to see who wins next week!

**Reply by parthiv11** [reply]
Glad to see my name in list.
Thank you for T-shirt! 💚💚

**Comment by chandrevdw31** [comment]
Thank you to everyone's continuous support! Here are this week's winners:

- Leonid Vinogradov https://github.com/leon-win
- Pär Thunberg https://github.com/barntarnst
- Szymon Łukaszewicz https://github.com/taurthil
- Borja Herrero https://github.com/bherrero
- Mena Heero https://github.com/mena-heero
- Mahmoud Zalt https://github.com/Mahmoudz
- dev-code0101 https://github.com/dev-code0101
- Ilya Noskov https://github.com/ilyanoskov
- Ovidiu Galatan https://github.com/galatanovidiu
- Nikki Johnson Russell https://github.com/wdtLMFO

If your name is mentioned above, please contact Chandre to claim your **limited edition** T-shirt.

We have one more week left of giving out T-shirts, so make sure to tune in next week!

**Comment by chandrevdw31** [comment]
**MindsDB is still celebrating reaching 20K stars!**

We are concluding giving out our **Limited Edition 20K Github Star T-shirts**! We thank everyone who has been on this journey with us and for your continuous support.

Here are the names of the winners:
- HT-Moh https://github.com/HT-Moh
- Karthick MJ https://github.com/karthickmj
- Luca Müller https://github.com/cuvar
- Wyatt Walsh https://github.com/wyattowalsh
- teocns https://github.com/teocns
- Ulukbek https://github.com/usamakov
- nappat87 https://github.com/nappat87
- Crystal https://github.com/Crystal-Cariaga
- Brian Roach https://github.com/itsbrex
- Kipruto https://github.com/kelvinkipruto

If your name is mentioned above, please contact Chandre on slack to claim your **limited edition** T-shirt.

We are extremely grateful,
The MindsDB Team 💚
